app/services/github/extract_repositories.rb

```diff
@@ -7,39 +7,68 @@ class ExtractRepositories
     def call
       response = HTTParty.get('https://raw.githubusercontent.com/he4rt/4noobs/master/README.MD')
 
-      extract_url_info(extract_links(response.body)).compact
+      extract_repositories_info(response.body).compact
     end
 
     private
 
-    # @param response_string [String] This represent the whole README
-    # @return [Array<String>]
-    def extract_links(response_string)
-      regex = %r{\[Clique aqui ➡️\]\((https://github\.com/.*?)\)}
+    # Returns an array of hashes representing GitHub repository information of a 4Noob.
+    #
+    # This method extracts each row from the README markdown tables, and iterates
+    # over each row to extract 4Noob repository information.
+    #
+    # @param response_string [String] This represent the whole raw README
+    # @return [Array<Hash>] An array of hashes containing GitHub repository information of a 4Noob.
+    def extract_repositories_info(response_string)
+      repositories_info = []
+
+      table_row_regex = %r{\|.*\|?}
```

Comment:
Você pode declarar um regex só colocando `//` em volta da expressão, dessa forma temos algo mais limpo:

```suggestion
      table_row_regex = /\|.*\|?/
```

Reply:
Sugiro que você execute `bundle exec rubocop -a` em seu projeto para corrigir essas pequenas práticas recomendadas para a linguagem

Vou configurar uma ação do github para executar isso no PR também :D